spec/models/user_model_spec.rb

```diff
@@ -47,6 +47,7 @@
 
   describe "ClassMethods" do
     before :all do
+      DatabaseCleaner.start
```

Comment:
Same as for profile spec, moving to spec helper will help to prevent repetition.